src/main/java/com/amihaiemil/versioneye/RtVersionEye.java

```diff
@@ -92,5 +92,10 @@ public VersionEye trusted() throws IOException {
             this.entry.through(TrustedWire.class)
         );
     }
+
+    @Override
+    public Me meApi() {
```

Comment:
@SherifWaly better name this simply ``me()`` and add an exception for checkstyle. You can make checkstyle ignore lines of your code if you add the annotation ``@checkstyle NameOfException (nr of lines to skip)``, like for example [here](https://github.com/jcabi/jcabi-github/blob/master/src/test/java/com/jcabi/github/DeployKeyTest.java#L41)